src/libs/Permissions.ts

```diff
@@ -35,8 +30,8 @@ function isBetaEnabled(beta: Beta, betas: OnyxEntry<Beta[]>): boolean {
 }
 
 export default {
-    canUseCustomReportNames,
     canUseLinkPreviews,
     isBlockedFromSpotnanaTravel,
     isBetaEnabled,
+    canUseAllBetas,
```

Comment:
I think this is actually not used anywhere outside of this file so we do not have to expose it
```suggestion
```